Cargo.toml

```diff
@@ -11,12 +11,13 @@ license = "MIT OR Apache-2.0"
 keywords = ["sessions", "tide", "async-session", "redis"]
 categories = ["web-programming::http-server", "web-programming", "database"]
 
-[dependencies.redis]
-version = "0.21.0"
-features = ["aio", "async-std-comp"]
-
 [dependencies]
 async-session = "3.0.0"
 
+[dependencies.redis]
+version = "0.21.0"
+features = ["aio", "async-std-comp", "connection-manager"]
```

Comment:
The Redis runtime comp feature flag needs to depend on a crate feature flag.

Reply:
> The Redis runtime comp feature flag needs to depend on a crate feature flag.

That'll be a separate PR if other runtimes are added.

Reply:
To get his PR merged, you need to introduce feature flags to allow this crate and its dependencies to be compiled either with tokio or with async-std.

The crate feature flags also need to toggle flags in the dependencies.

Reply:
That is not relevant to this PR. Tokio is #23. This only adds connection multiplexing.